fasto/Fasto/Interpreter.fs

```diff
@@ -146,8 +147,16 @@ let rec evalExp (e : UntypedExp, vtab : VarTable, ftab : FunTable) : Value =
   *)
   | Times(_, _, _) ->
         failwith "Unimplemented interpretation of multiplication"
-  | Divide(_, _, _) ->
-        failwith "Unimplemented interpretation of division"
+  | Divide(e1, e2, pos) ->
+        let (res1, res2) = (evalExp(e1, vtab, ftab), evalExp(e2, vtab, ftab))
+        match (res1, res2) with
+            | (IntVal a, IntVal b) -> 
+              try
+                IntVal(a/b)
+              with
+                | :? DivideByZeroException -> reportDivByZero (expPos e2)  
+            | (IntVal _, _) -> reportWrongType "right operand of /" Int res2 (expPos e2)
+            | ( _, _) -> reportWrongType "left operand of /" Int res1 (expPos e1)
```

Comment:
I'm not sure if it could pay off to also include an error message, when both are false. Since it would ReportWrongType with message "left operand of /", with one being the right type and with both being the wrong type.

a/2 -> err2
a/b -> err2

Reply:
For example:
```
            | (IntVal _, _) -> reportWrongType "right operand of /" Int res2 (expPos e2)
            | (_, IntVal _) -> reportWrongType "left operand of /" Int res1 (expPos e1)
            | ( _, _) -> reportWrongType "all operands of /" Int res1 (expPos e1)
```

Reply:
Yeah exactly, what I thought about, but you properly know best if it's better to be so specific or not, because ether way an error gets shown and one is likely to know what it is from ether the error messages.